tests/cover/test_explicit_examples.py

```diff
@@ -162,6 +162,20 @@ def test_positive(x):
     assert u'Falsifying example: test_positive(x=-1)' in out
 
 
+def test_prints_verbose_output_for_explicit_examples():
+    @settings(verbosity=Verbosity.verbose)
+    @example('NOT AN INTEGER')
+    @given(integers())
+    def test_positive(x):
```

Comment:
nit: `test_positive` isn’t really an accurate name for this function, is it?

Maybe `test_always_passes`? Or `test_is_not_none`? Just so this isn't inaccurate.